content/custom-commands/commands.md

```diff
@@ -239,6 +239,38 @@ You must specify a channel to run interval commands in even if the command doesn
 
 {{% /notice %}}
 
+#### Component
+
+{{% notice style="tip" %}}
+
+Use of interactions within YAGPDB is an advanced topic, the documentation should be used only as reference. To learn
+about using interactions, [see here](/reference/custom_interactions).
+
+{{% /notice %}}
+
+The component trigger is used to trigger custom commands via buttons or select menus.
+
+When creating a button or a select menu using template scripting, you must define a `custom_id`. When that button/menu
+is used by a member of your server, it triggers any custom command whose trigger matches that Custom ID.
```

Comment:
```suggestion
```
I would just leave this sentence out entirely. It is, in spirit, a Write
Everything Twice (WET). You already explained in the cookbook that a
custom ID must be defined for a component to be able to trigger a custom
command.

Perhaps just hint at a custom_id regex being required, if this suggestion
is too DRY for you.